### PR TITLE
Add test for cmd/kubeadm/app/phases/upgrade/policy.go

### PR DESCRIPTION
**Pull request by my-git9**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Add test for cmd/kubeadm/app/phases/upgrade/policy.go

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/phases/upgrade/policy_test.go [hunk 1]
```diff
@@ -231,3 +231,74 @@ func TestEnforceVersionPolicies(t *testing.T) {
 		})
 	}
 }
+
+func Test_detectUnstableVersionError(t *testing.T) {
```
Comment (SataQiu) [inline cmd/kubeadm/app/phases/upgrade/policy_test.go:235]: Thanks @my-git9 
But it seems that the above `TestEnforceVersionPolicies` has covered the test cases about detectUnstableVersionError.

Comment (my-git9) [reply cmd/kubeadm/app/phases/upgrade/policy_test.go:235]: Thanks for reminding, I will close it

**Pull request closed without merging**